samples/nfc/tag_reader/src/main.c

```diff
@@ -43,6 +57,180 @@ static const struct st25r3911b_nfca_buf rx_buf = {
 	.len = sizeof(rx_data)
 };
 
+enum {
+	T2T_IDLE,
+	T2T_HEADER_READ,
+	T2T_DATA_READ
+};
+
+struct t2t_tag {
+	u32_t state;
```

Comment:
Can we use enum as a type for state? (you need to name enum type)

or

combine enum declaration with state field definition within the scope of the structure

Reply:
Ok